src/compiler/emitter.ts

```diff
@@ -3303,6 +3313,21 @@ module ts {
                 decreaseIndent();
             }
 
+			function emitSignatureParametersES6(node: FunctionLikeDeclaration) {
+				// Check the node's parameters whether it contains flags indicating that it has no parenthesis around the parameters
+				// Preserver no-parenthesis
+				if (node && node.flags & NodeFlags.SimpleArrowFunction) {

```

Comment:
Agree with you on that. I just check to make sure but you are right this shouldn't happen